tests/specs/import_map/import_map_config/__test__.json

```diff
@@ -0,0 +1,29 @@
+{
+  "httpServer": true,
+  "steps": [{
+    "args": "run --quiet --reload --import-map=import_map.json test.ts",
+    "output": "run.out"
+  }, {
+    "args": "run --quiet --reload --import-map=import_map_invalid.json --config=config.json test.ts",
+    "output": "flag_has_precedence.out",
+    "exitCode": 1
+  }, {
+    "args": "run --reload --config=config.json test.ts",
+    "output": "config.out"
+  }, {
+    "clean": true,
```

Comment:
Since the test sets up temp directory once then runs all these commands, I needed a way to "clean" the deno_dir when certain steps get run so that it goes and does stuff again likes downloads packages/remote deps.

Reply:
Can you clarify? Do we spawn one-process-per-step in a shared temporary directory acting as CWD are all these tests run in a single process?

`clean` options feels a bit magical, what do you think about using array notation for this file and moving tests that use `clean` to separate objects? Feels like one fewer concept to understand.

Reply:
I renamed it to `cleanDenoDir: true`. What do you mean by using array notation?

> Do we spawn one-process-per-step in a shared temporary directory acting as CWD are all these tests run in a single process?

We spawn one process per step, but they all share the same temporary deno_dir. If we had a `deno clean` command then I could run that as a separate step between the steps in order to clean the deno_dir.

Reply:
Okay, that is fine.

I meant to create and array and put each step into a separate object, that would recreate temp dir for each test, but I guess it's slower in this case.

Reply:
Oh, that's not supported. Right now it needs to be either a single step at the top level or multiple steps in an array under "steps": https://github.com/denoland/deno/pull/22877/files#diff-35963c548632182cdbc8f4d58579ffbb68c7e2c513c7deefb504fff228207265R29